suite-native/biometrics/src/components/BiometricOverlay.tsx

```diff
@@ -13,10 +16,26 @@ const overlayWrapperStyle = prepareNativeStyle(utils => ({
 
 export const BiometricOverlay = () => {
     const { applyStyle } = useNativeStyles();
+    const { translate } = useTranslate();
+    const { isBiometricsAuthenticationCanceled, setIsBiometricsAuthenticationCanceled } =
+        useIsBiometricsAuthenticationCanceled();
+
+    const handleReenable = () => {
+        // Setting this to true lets useBiometrics to ask for biometrics in case it was canceled by user before
+        // https://github.com/trezor/trezor-suite/issues/10647
+        setIsBiometricsAuthenticationCanceled(false);
+    };
 
     return (
         <Box style={applyStyle(overlayWrapperStyle)}>
```

Comment:
Redundant box now. Please remove it and apply the styles to VStack so the content is centered.